lib/AST/PrettyStackTrace.cpp

```diff
@@ -209,6 +210,15 @@ void swift::printTypeDescription(llvm::raw_ostream &out, Type type,
   if (addNewline) out << '\n';
 }
 
+void PrettyStackTraceClangType::print(llvm::raw_ostream &out) const {
+  out << "While " << Action << ' ';
+  if (TheType == nullptr) {
+    out << "NULL clang type!\n";
+    return;
+  }
+  TheType->dump(out);
```

Comment:
This incurs a non-header-only dependency on the Clang AST/Type.h header in our AST. The "solution" I'd used earlier was to pass in an ASTContext and grab a ClangModuleLoader off that. I'm going to submit the PR today or tomorrow to re-land the Clang types in SIL, so if this is getting merged after that, you can use the same helper method.

Reply:
Per offline discussion, this is okay.